### DISCUSSION TITLE
CSS not showing

### DISCUSSION THREAD

**Discussion by rndadhdman** [Kovah/LinkAce#527] [Support]
I have LinkAce on a sub-directory on my main domain. the .env app_url reflects this. The site is up and running. I am able to log into the site. All checks passed. However, I am not seeing the CSS loading. 

![2022-08-23 15_30_46-](https://user-images.githubusercontent.com/13965470/186249708-ff0244da-a6f7-46dc-88d8-0f69886a69d6.png)

Site Config:
 
```
Alias "/links" "/var/www/html/linkace/public"

    <Directory "/var/www/html/linkace/public">
      Options FollowSymlinks
      AllowOverride None
      Require all granted

      <IfModule mod_rewrite.c>
          <IfModule mod_negotiation.c>
                Options -MultiViews -Indexes
          </IfModule>
      </ifModule>

        RewriteEngine On

        # Handle Authorization Header
        RewriteCond %{HTTP:Authorization} .
        RewriteRule .* - [E=HTTP_AUTHORIZATION:%{HTTP:Authorization}]

        # Redirect Trailing Slashes If Not A Folder...
        RewriteCond %{REQUEST_FILENAME} !-d
        RewriteCond %{REQUEST_URI} (.+)/$
        RewriteRule ^ %1 [L,R=301]

        # Handle Front Controller...
        RewriteCond %{REQUEST_FILENAME} !-d
        RewriteCond %{REQUEST_FILENAME} !-f
        RewriteRule ^ index.php [L]

    </Directory>
```

File permissions:

```
chmod -R 755 /var/www/html/linkace/
chown -R www-data:www-data /var/www/html/linkace/
```

File paths are named as above. Here is the file tree. I can see the CSS code. 

![2022-08-23 15_28_44-Enroll devices - Microsoft Endpoint Manager admin center](https://user-images.githubusercontent.com/13965470/186249459-1b8c2503-560a-45e2-ac79-1ef194b24288.png)

.env file

```
## LINKACE CONFIGURATION

## Basic app configuration
# The app key is generated later, please leave it blank
APP_KEY=base64:blahblahblah

APP_URL=https://mysite.com/links

## Configuration of the database connection
## Attention: Those settings are configured during the web setup, please do not modify them now.
# Set the database driver (mysql, pgsql, sqlsrv)
DB_CONNECTION=mysql
# Set the host of your database here
DB_HOST=localhost
# Set the port of your database here
DB_PORT=3306
# Set the database name here
DB_DATABASE=blahblahblah
# Set both username and password of the user accessing the database
DB_USERNAME=blahblahblah_user
DB_PASSWORD=blahblahblahblahblahblahblahblahblah
```

PHP VERSION: 8.1.2
LinkAce: 1.10.3

**Comment by Kovah** [comment]
Hi! Thanks for your detailed report.
Installation in a sub directory is currently not officially supported. See #503.

You could [try to make some changes](https://stackoverflow.com/questions/50056229/laravel-application-on-sub-directory) to the app, but I do not recommend this as it may cause unknown side effects.
